modules/orbit/andes/pom.xml

```diff
@@ -163,6 +163,7 @@
                         <Include-Resource>src/main/resources</Include-Resource>
                         <Bundle-Activator>org.wso2.andes.wso2.internal.QpidBundleActivator
                         </Bundle-Activator>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
yes it is discouraged if we are not allowing custom classes to be loaded. Since we have the configuration in broker.xml to load custom authenticator for mqtt, we should allow the class loader to able to load any classes.